utils/cryptography/encryption.go

```diff
@@ -27,7 +27,7 @@ func EncryptMessage(encryptionKey [32]byte, field []byte) (encrypted []byte, non
 
 	// large amounts of data should be chunked
 	// If in doubt, 16KB is a reasonable chunk size.
-	if len(field) >= (16 * 1000) {
+	if len(field) >= (4 * 1000) {
```

Comment:
(1) Either code or comment should be updated. 
(2) Prefer making this a constant over magic number

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/816/commits/e2da4ad81cf34ec855abf2402ea3eab28a3ae0f7